src/main/webapp/content/css/global.css

```diff
@@ -231,3 +231,10 @@ Social Buttons
     color: #fff;
 }
 /* jhipster-needle-css-add-main JHipster will add new css style */
+
+/* ==========================================================================
+Create Study Modal
+========================================================================== */
+.email-body{
```

Comment:
this class doesn't target anything? Maybe you meant `emailBody`

Reply:
Fixed thanks.

Reply:
I don't think you've pushed up these changes